cmd/glasskube/cmd/describe.go

```diff
@@ -154,6 +206,21 @@ func printRepositories(pkg *v1alpha1.Package, repos []v1alpha1.PackageRepository
 	}
 }
 
+func repositoriesAsMap(pkg *v1alpha1.Package, repos []v1alpha1.PackageRepository) []map[string]string {
+	repositories := []map[string]string{}
+	for _, repo := range repos {
+		repository := make(map[string]string)
+		repository["Name"] = repo.Name
+		if isInstalledFrom(pkg, repo) {
+			repository["Installed"] = "true"
+		} else {
+			repository["Installed"] = "false"
+		}
```

Comment:
Please start the property names with lowercase characters.

Also, I think it would be better to only set this property if `isInstalledFrom` returns true (i.e. removing the `else` branch).